### ISSUE TITLE
NullPointerException in io.crate.analyze.symbol.SymbolVisitor.process

### ISSUE THREAD

**Issue by DLT1412** [crate/crate#6478]
**CrateDB version**:
2.1.8
**JVM version**:
java version "1.8.0_92"
Java(TM) SE Runtime Environment (build 1.8.0_92-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.92-b14, mixed mode)
**OS version / environment description**:
CentOS 7
**Problem description**:

`SQLActionException[NullPointerException in io.crate.analyze.symbol.SymbolVisitor.process(SymbolVisitor.java:32)]`

**Steps to reproduce**:
```
create table j1 (id1 int, id2 int, id3 int, id4 int, id5 int) clustered into 1 shards;
create table j2 (id name) clustered into 1 shards;
create table j3 (id name) clustered into 1 shards;
create table j4 (id name) clustered into 1 shards;
insert into j1 values (1, 1, 1, 1);
insert into j2 (id, name) values (1, 'name1');
insert into j3 (id, name) values (1, 'name1');
insert into j4 (id, name) values (1, 'name1');
select id1 from (select id1, id2, id3, id4 from j1) a left join j2 b on a.id2 = b.id left join j3 c on a.id3 = c.id  left join j4 d on a.id4 = d.id;
```

**Use case**:
Select specific some column when join multi-table
Note: select id1->id3 is error, select * or id4 is ok.
**Feature description**:

**Comment by mxm** [comment]
Thanks for reporting! The issue has already been fixed for the upcoming 2.3.0 release. We are working on a fix for older versions.

**Comment by mxm** [comment]
The issue is fixed for the upcoming release: 2.1.9, 2.2.2, and 2.3.0.
